### ISSUE TITLE
Browse mode fail in Chrome 78 for Portable NVDA

### ISSUE THREAD

**Issue by larry801** [nvaccess/nvda#10423]
### Steps to reproduce:
Open Google Chrome (78.0.3904.70 or higer)
Go to https://www.github.com
Press NVDA+Space twice switch back to browse mode
Press H
### Actual behavior:
NVDA do not respond
### Expected behavior:
Jump to next heading
#### NVDA installed/portable/running from source:
Portable 
#### NVDA version:
2019.2.1
#### Windows version:
Windows 8.1 
#### Name and version of other software in use when reproducing the issue:
Chrome 78.0.3904.70
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes alpha 19012 also have this issue
Here is  the error log on alpha
ERROR - scriptHandler.executeScript (16:25:42.206) - MainThread (2476):
error executing script: <bound method GlobalCommands.script_toggleVirtualBufferPassThrough of <globalCommands.GlobalCommands object at 0x08E67410>> with gesture 'NVDA+空格'(NVDA+Space)
Traceback (most recent call last):
  File "scriptHandler.pyc", line 205, in executeScript
  File "globalCommands.pyc", line 1305, in script_toggleVirtualBufferPassThrough
  File "treeInterceptorHandler.pyc", line 139, in _set_passThrough
  File "review.pyc", line 124, in setCurrentMode
  File "review.pyc", line 48, in getDocumentPosition
  File "documentBase.pyc", line 24, in makeTextInfo
  File "virtualBuffers\__init__.pyc", line 207, in __init__
  File "textInfos\offsets.pyc", line 416, in __init__
  File "virtualBuffers\__init__.pyc", line 195, in _getOffsetsFromNVDAObject
  File "virtualBuffers\__init__.pyc", line 190, in _getOffsetsFromNVDAObjectInBuffer
  File "virtualBuffers\__init__.pyc", line 164, in _getOffsetsFromFieldIdentifier
OSError: [WinError 1775] 在远程过程调用时，将空的上下文句柄从客户端传递到主机。
A null context handle was passed from the client to the host during a remote procedure call
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (16:27:43.784) - MainThread (2476):
Developer info for navigator object:
name: 'Error with NVDA next'
role: ROLE_DOCUMENT
roleText: None
states: STATE_READONLY, STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.IAccessible.chromium.Document object at 0x07281D10>
Python class mro: (<class 'NVDAObjects.IAccessible.chromium.Document'>, <class 'NVDAObjects.IAccessible.ia2Web.Document'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'object'>)
description: None
location: RectLTWH(left=0, top=133, width=1024, height=635)
value: None
appModule: <'appModuleHandler' (appName 'chrome', process ID 17252) at address 630ca10>
appModule.productName: 'Google Chrome'
appModule.productVersion: '78.0.3904.70'
TextInfo: <class 'NVDAObjects.IAccessible.IA2TextTextInfo'>
windowHandle: 661962
windowClassName: 'Chrome_RenderWidgetHostHWND'
windowControlID: -1060616896
windowStyle: 1445986304
extendedWindowStyle: 32
windowThreadID: 19360
windowText: 'Chrome Legacy Window'
displayText: ''
IAccessibleObject: <POINTER(IAccessible2) ptr=0x66867c4 at 6edb3a0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=661962, objectID=-4, childID=-189747
IAccessible accName: 'Error with NVDA next'
IAccessible accRole: ROLE_SYSTEM_DOCUMENT
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_READONLY, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048644)
IAccessible accDescription: None
IAccessible accValue: 'https://nvda.groups.io/g/nvda/topic/error_with_nvda_next/8683800?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,40,8683800'
IAccessible2 windowHandle: 661962
IAccessible2 uniqueID: -189747
IAccessible2 role: ROLE_SYSTEM_DOCUMENT
IAccessible2 states: IA2_STATE_OPAQUE (1024)
IAccessible2 attributes: 'tag:#document;explicit-name:true;'

**Comment by larry801** [comment]
This issue does not happen in an installed version.

**Comment by dingpengyu** [comment]
cc @michaelDCurran 
and
@feerrenrut 
thank

**Comment by larry801** [comment]
If run from source , this issue is still reproducible.

**Comment by XLTechie** [comment]
I can definitely reproduce this in latest alpha (19012) on Windows 10 1903.

-- 
Luke Davis
Moderator: the new NVDA Help mailing list! (https://groups.io/g/NVDAHelp)
Author: Debug Helper NVDA add-on (https://addons.nvda-project.org/addons/debugHelper.en.html)

**Comment by Adriani90** [comment]
@larry801, @XLTechie can you still reproduce this issue in Chrome 81 and NVDA 2020.1?

**Comment by larry801** [comment]
Cannot reproduce in alpha 20200 Chrome 81 windows 8.1
